content/topics/intro-to-typescript/_index.md

```diff
@@ -5,11 +5,18 @@ ready: true
 title: Introduction to Typescript
 ---
 
+### What is Typescript
+
+- [TypeScript](https://flaviocopes.com/typescript/) is an extension of JavaScript that adds types as well as other language features. 
+- The [TypeScript compiler](https://code.visualstudio.com/docs/typescript/typescript-compiling) “tsc” compiles TypeScript programs into JavaScript programs and in the process analyzes them and reports type errors. 
```

Comment:
You introduce Typescript and then immediately introduce the TypesScript compiler and then you continue to talk about typescript in the next point, Is there a way we can make this can flow a bit better. Say everything you need to say about Typescript, and then you can say what you need to about the compiler